mod-configuration-server/src/test/java/org/folio/rest/RestVerticleTest.java

```diff
@@ -1395,45 +1356,44 @@ public void checkURLs(TestContext context) {
 
   private void createSampleRecords(TestContext context) {
     try {
-      //save config entry
-      String sample = getFile("kv_configuration.sample");
-
-      ConfigurationRecordBuilder baselineFromSample = ConfigurationRecordBuilder.from(sample);
-
+      JsonObject configRecord = new ConfigurationRecordBuilder()
+        .withModuleName("DUMMY")
+        .withDescription("dummy module for testing")
+        .withConfigName("dummy_rules")
+        .withCode("config_data")
+        .withValue("")
+        .create();
+
+      configRecord.put("default", "true");
```

Comment:
Is there no `withDefault` method?

Reply:
There is not.  I looked.  I can only speculate about why it was excluded.  Perhaps because none of the other tests seem to need to set it, and this test, which is the only one that does, was pulling data from an external file?

Reply:
Would extending the `ConfigurationRecordBuilder` to include a `withDefault` method mean some of these steps could be avoided?

Reply:
I would be able to avoid setting the default property using the "put" method, but I don't think it impacts anything else.  I still need a JSONOject to feed to "with()", so I'm still going to be doing the funky "build this object then rebuild it."  This is part of why I didn't do so.  The other reason is that the builder is full of overloaded constructors, all of which would need to be changed.  Would you like me to do that instead?

Reply:
If the builder always return a copy (which I'm guessing it does from your description of the constructors) then it should be sufficient to use a builder without having initialised it from JSON as the starting point.

At which point, I think the `build this object then rebuild it` becomes unnecessary.

Reply:
So, try to create an "empty" builder object, then use its methods to create the record structure in place, then use that as the baseline record?  That is simpler, if it works.  I'll give it a try.

Reply:
Latest push implements this.

Reply:
Has the code been pushed, github is still reporting the previous code to me?

Reply:
Try it now.  I thought I had pushed it, apparently not.